### DISCUSSION TITLE
Reference Points in MOOSE

### DISCUSSION THREAD

**Discussion by Flolaffel** [idaholab/moose#20396] [Q&A General] [answered]
Is there a way to create a reference point in MOOSE which I can use to apply the load for my model?

Thanks in advance.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Sorry we missed this. Yes you can create a nodeset using this
https://mooseframework.inl.gov/source/meshgenerators/ExtraNodesetGenerator.html
then you should apply the load on that nodeset

Guillaume
